asapdiscovery-data/asapdiscovery/data/schema/ligand.py

```diff
@@ -215,13 +215,30 @@ def from_oemol(cls, mol: oechem.OEMol, **kwargs) -> "Ligand":
         keys_to_save = [
             key for key in kwargs.keys() if key not in cls.__fields__.keys()
         ]
-        tags = {(key, value) for key, value in kwargs.items() if key in keys_to_save}
+
+        tags = set()
```

Comment:
if you want to I think you could put this back as a list comprehension with `isinstance(..., collections.abc.Hashable)` but this is fine too, up to you